pipelines/_ingredients/LSSTCam/DRP.yaml

```diff
@@ -75,7 +75,8 @@ tasks:
     class: lsst.analysis.tools.tasks.refCatSourceAnalysis.RefCatSourceAnalysisTask
     config:
       # Only run metrics for analyzing the preSources:
-      connections.data: preSourceTable_visit_gaia_dr3_20230707_match_astrom
+      connections.data: preSourceTable_visit_uw_stars_20240524_match_astrom
```

Comment:
I think what was already here was correct. The `uw_stars...` catalog is only for ComCamSim. (The outputName below looks correct however.)

Reply:
I think this was changed in an effort to diagnose the cascade of errors. Changed back, here and for LSSTComCam.